src/main/java/gameframework/gui/GameWindow.java

```diff
@@ -32,11 +62,39 @@ public GameWindow(String gameName, GameCanvas gameCanvas,
 		this.statusBar.addAll(elementsStatusBar);
 		this.frame = new Frame(gameName);
 		this.gameCanvas = gameCanvas;
-		this.gameCanvas.setSize(//
-				configuration.getSpriteSize() * configuration.getNbColumns(), //
+		this.gameCanvas.setSize(
+				configuration.getSpriteSize() * configuration.getNbColumns(), 
 				configuration.getSpriteSize() * configuration.getNbRows());
 	}
+	
+	/**
+	 * hide the status bar
+	 */
+	public void hideStatusBar() {
+		showStatusBar(false);
+	}
+	
+	/**
+	 * show the status bar
+	 */
+	public void showStatusBar() {
+		showStatusBar(true);
+	}
+	
+	/**
+	 * Shows or hide the status bar depending of the value of parameter showStatusBar.
+	 * @param showStatusBar if true - shows the status bar; otherwise, hides the status bar.
+	 */
+	private void showStatusBar(boolean showStatusBar) {
```

Comment:
please replace `private` by `protected` everywhere, you never know what your subclasses will need to do.